scripts/dudle.coffee

```diff
@@ -54,7 +54,7 @@ module.exports = (robot) ->
     robot.respond /dudlelist/i, (msg) ->
         msg.send 'Aktuell tracke ich folgende Dudles:'
         dudles = read_dudles_file()
-        for dudle in dudles
+        dudles.forEach (dudle) ->
```

Comment:
Any reason for this? Just curious. For loops in coffeescript are pretty cool as they actually do what one would think in stark contrast to plain JS.